packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3457,22 +3457,34 @@ describe('plugin-meetings', () => {
         sandbox = null;
       });
 
-      describe('#stopFloorRequest', () => {
-        it('should have #stopFloorRequest', () => {
-          assert.exists(meeting.stopFloorRequest);
+      describe('#releaseScreenShareFloor', () => {
+        it('should have #releaseScreenShareFloor', () => {
+          assert.exists(meeting.releaseScreenShareFloor);
         });
         beforeEach(() => {
-          meeting.locusInfo.mediaShares = [{name: 'content', url: url1}];
+          meeting.selfId = 'some self id';
+          meeting.locusInfo.mediaShares = [{name: 'content', url: url1, floor: {beneficiary: {id: meeting.selfId}}}];
           meeting.locusInfo.self = {url: url2};
+          meeting.mediaProperties = {mediaDirection: {sendShare: true}};
           meeting.meetingRequest.changeMeetingFloor = sinon.stub().returns(Promise.resolve());
         });
-        it('should call change meeting floor', async () => {
-          const share = meeting.share();
+        it('should call changeMeetingFloor()', async () => {
+          const share = meeting.releaseScreenShareFloor();
```

Comment:
this test has always been wrong - it was calling share() instead of stopFloorRequest(), so was never testing what it was supposed to test. After changing this line I had to add a couple more things to beforeEach() (see above) to make it work.